src/wcl/tracing.cpp

```diff
@@ -174,24 +154,5 @@ void FormatSubscriber::receive(const Event& e) {
   }
 }
 
-void FatalEventSubscriber::receive(const Event& e) {
```

Comment:
This better version of this comes back with https://github.com/sifive/wake/pull/1284